packages/backend-core/src/cache/user.ts

```diff
@@ -27,6 +28,47 @@ async function populateFromDB(userId: string, tenantId: string) {
   return user
 }
 
+async function populateUsersFromDB(
+  userIds: string[],
+  tenantId: string
+): Promise<{ users: User[]; notFoundIds?: string[] }> {
+  const db = tenancy.getTenantDB(tenantId)
+  const allDocsResponse = await db.allDocs<any>({
```

Comment:
if we could add this to the user DB functionality (`UserDB`) as a function for fetching bulk users, that would be good. There might even be one there already!

Reply:
Good shout, I missed that db utils :) This simplified my tests a lot